lib/tests/formulae_detect.rb

```diff
@@ -71,6 +71,11 @@ def detect_formulae!(args:)
             origin_ref = "origin/#{base_ref}"
             diff_start_sha1 = rev_parse(origin_ref)
             diff_end_sha1 = github_sha
+          # Use GitHub Actions variables for merge group jobs.
+          elsif ENV.fetch("GITHUB_EVENT_NAME", nil) == "merge_group"
+            origin_ref = "origin/#{github_ref.gsub(%r{^refs/heads/}, "")}"
+            diff_end_sha1 = github_sha
+            diff_start_sha1 = rev_parse("#{github_sha}^")
```

Comment:
```suggestion
            diff_start_sha1 = rev_parse(origin_ref)
            diff_end_sha1 = github_sha
```

I think this should be correct?

Reply:
It may work out to be the same, since I haven't found a case where `github_sha^` wasn't `origin/master`. But let's try this.